hlx_statics/blocks/header/header.js

```diff
@@ -2,20 +2,530 @@ import {
   createTag,
   setActiveTab,
   focusRing,
-  isLocalHostEnvironment,
   isTopLevelNav,
-  setSearchFrameOrigin,
   getClosestFranklinSubfolder,
-  setQueryStringParameter,
-  getQueryString,
 } from '../../scripts/lib-adobeio.js';
 import { readBlockConfig, getMetadata, fetchTopNavHtml } from '../../scripts/lib-helix.js';
 import { loadFragment } from '../fragment/fragment.js';
 
+const ALGOLIA_CONFIG = {
+  APP_KEY: window.adp_search.APP_KEY || '',
```

Comment:
Noticing a mix of using `_` and camel case in variable names. Please use camel case at much as possible unless it's an env var like this then `_` is ok.